demo/scripts/demoRelease.js

```diff
@@ -89,10 +72,6 @@ function tagAndPublish(newVersion) {
   console.log(`trying to publish ${newVersion}...`);
   exec.execSync(`npm --no-git-tag-version version ${newVersion}`);
   exec.execSync(`npm publish --tag ${VERSION_TAG}`);
-  if (isRelease) {
-    exec.execSync(`git tag -a ${newVersion} -m "${newVersion}"`);
-  }
-  exec.execSyncSilent(`git push deploy ${newVersion} || true`);
```

Comment:
Don't we need that? for git tags?

Reply:
this is demoRelease.js. Github slightly messed files